examples/stable/mssql/mssql-blueprint.yaml

```
@@ -0,0 +1,85 @@
+apiVersion: cr.kanister.io/v1alpha1
+kind: Blueprint
+metadata:
+  name: mssql-blueprint
+  namespace: kanister
+actions:
+  backup:
+    outputArtifacts:
+      mssqlCloudDump:
+        keyValue:
+          s3path: "{{ .Phases.dumpToObjectStore.Output.s3path }}"
+    secretNames:
+      - mssql
+    phases:
+      - func: KubeTask
+        name: dumpToObjectStore
+        args:
+          image: chaitanyadelphian/mssql-tools:1.0.2
```

```suggestion
          image: ghcr.io/kanisterio/mssql-tools:0.71.0
```
Let's fix this once done with testing